assets/auth.js

```diff
@@ -0,0 +1,3 @@
+import { mount } from 'apps/authentication/client';
+
+$(mount);
```

Comment:
a simple thing, but where possible let's get rid of jquery:

```javascript
document.addEventListener('DOMContentLoaded', mount);
```

Reply:
👍